### PR TITLE
注文時に注文日次と合計金額を記録するように修正しました

### PR DESCRIPTION
**Pull request by SekiMoemi**
ご確認お願いします。

### CODE REVIEW

File: src/main/java/jp/co/example/ecommerce_a/service/OrderService.java [hunk 1]
```diff
@@ -36,8 +37,11 @@ public void order(OrderForm form) throws ParseException {
 		Order targetOrder = orderRepository.load(form.getIntOrderId());
 
 		BeanUtils.copyProperties(form, targetOrder);
+		Date date = new Date();
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/service/OrderService.java:40]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/service/OrderService.java:40]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def' child has incorrect indentation level 16, expected level should be 4.

File: src/main/java/jp/co/example/ecommerce_a/service/OrderService.java [hunk 2]
```diff
@@ -36,8 +37,11 @@ public void order(OrderForm form) throws ParseException {
...
+		targetOrder.setOrderDate(date);
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/service/OrderService.java:41]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/service/OrderService.java:41]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def' child has incorrect indentation level 16, expected level should be 4.

File: src/main/java/jp/co/example/ecommerce_a/service/OrderService.java [hunk 3]
```diff
@@ -36,8 +37,11 @@ public void order(OrderForm form) throws ParseException {
...
 		targetOrder.setDeliveryTime(form.getTimeStampDeliveryTime());
 		targetOrder.setPaymentMethod(form.getIntPaymentMethod());
+		targetOrder.setTotalPrice(targetOrder.getCalcTotalPrice()+targetOrder.getTax());
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/service/OrderService.java:44]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/service/OrderService.java:44]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def' child has incorrect indentation level 16, expected level should be 4.

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/service/OrderService.java:44]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.WhitespaceAroundCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>WhitespaceAround: '+' is not followed by whitespace. Empty blocks                may only be represented as {} when not part of a multi-block statement (4.1.3)

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/service/OrderService.java:44]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.WhitespaceAroundCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>WhitespaceAround: '+' is not preceded with whitespace.

**Review by yamada-kensyu — APPROVED** [review]
ありがとうございます！

**Pull request merged**